src/main/java/net/sf/jabref/logic/importer/fetcher/IsbnViaChimboriFetcher.java

```diff
@@ -0,0 +1,87 @@
+package net.sf.jabref.logic.importer.fetcher;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.List;
+import java.util.Optional;
+
+import net.sf.jabref.logic.importer.FetcherException;
+import net.sf.jabref.logic.importer.ImportFormatPreferences;
+import net.sf.jabref.logic.importer.ParseException;
+import net.sf.jabref.model.entry.BibEntry;
+
+import com.mashape.unirest.http.HttpResponse;
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import org.jsoup.helper.StringUtil;
+
+/**
+ * Fetcher for ISBN using https://bibtex.chimbori.com/, which in turn uses Amazon's API.
+ */
+public class IsbnViaChimboriFetcher extends AbstractIsbnFetcher {
+
+    public IsbnViaChimboriFetcher(ImportFormatPreferences importFormatPreferences) {
+        super(importFormatPreferences);
+    }
+
+    /**
+     * @return null, because the identifier is passed using form data. This method is not used.
+     */
+    @Override
+    public URL getURLForID(String identifier) throws URISyntaxException, MalformedURLException, FetcherException {
+        return null;
+    }
+
+    @Override
+    public Optional<BibEntry> performSearchById(String identifier) throws FetcherException {
+        if (StringUtil.isBlank(identifier)) {
+            return Optional.empty();
+        }
+
+        this.ensureThatIsbnIsValid(identifier);
+
+        HttpResponse<String> postResponse;
+        try {
+            postResponse = Unirest.post("https://bibtex.chimbori.com/isbn-bibtex")
```

Comment:
We should rewrite IdBasedParserFetcher so that it also supports POST actions (and maybe uses Unirest for it).